src/OpenSage.Game/Gui/Apt/ActionScript/Opcodes/Basic.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace OpenSage.Gui.Apt.ActionScript.Opcodes
+{
+    //declare a pool of constants that will be used in the following code
```

Comment:
In general, for this type of comment on classes and methods, you can use doc comments, so that they'll show up in IntelliSense:

``` csharp
/// <summary>
/// Declare a pool...
/// </summary>
public sealed class ConstantPool
```